test/smoke/testApps/Micrometer/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/MicrometerDisabledTest.java

```diff
@@ -0,0 +1,19 @@
+package com.microsoft.applicationinsights.smoketest;
+
+import org.junit.*;
+
+import static org.junit.Assert.*;
+
+@UseAgent("disabled_micrometer")
+public class MicrometerDisabledTest extends AiSmokeTest {
+
+    @Test
+    @TargetUri("/test")
+    public void doMostBasicTest() throws Exception {
+        mockedIngestion.waitForItems("RequestData", 1);
+
+        // sleep a bit and make sure no micrometer metrics are reported
+        Thread.sleep(10000);
```

Comment:
why here needs to wait 2.x longer? it's 5000 elsewhere.

Reply:
metrics are only reported every 5 seconds, and then batching can take another 5 seconds